infra/src/main/kotlin/fi/fta/geoviite/api/frameconverter/v1/FrameConverterControllerV1.kt

```diff
@@ -1,13 +1,84 @@
 package fi.fta.geoviite.api.frameconverter.v1
 
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.module.SimpleModule
 import fi.fta.geoviite.api.aspects.GeoviiteIntegrationApiController
-import org.springframework.web.bind.annotation.GetMapping
+import fi.fta.geoviite.api.frameconverter.geojson.GeoJsonFeature
+import fi.fta.geoviite.api.frameconverter.geojson.GeoJsonFeatureCollection
+import fi.fta.geoviite.infra.common.MainLayoutContext
+import fi.fta.geoviite.infra.error.IntegrationApiException
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder
+import org.springframework.web.bind.annotation.ModelAttribute
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestMethod
+import org.springframework.web.bind.annotation.RequestParam
 
-@GeoviiteIntegrationApiController("/frame-converter/v1")
-class FrameConverterControllerV1 {
+@Configuration
+class FrameConverterApiObjectMapperV1 {
 
-    @GetMapping("/some-conversion")
-    fun someConversion(): String {
-        return "some response"
+    @Bean
+    fun objectMapper(builder: Jackson2ObjectMapperBuilder): ObjectMapper {
+        val module = SimpleModule()
+        module.addDeserializer(FrameConverterRequestV1::class.java, FrameConverterRequestDeserializerV1())
+        return builder.build<ObjectMapper>().registerModule(module)
+    }
+}
+
+@GeoviiteIntegrationApiController(
+    [
+        "/rata-vkm",
+        "/rata-vkm/dev",
+        "/rata-vkm/v1",
+        "/rata-vkm/dev/v1",
+    ],
+)
+class FrameConverterControllerV1 @Autowired constructor(
```

Comment:
Mmh, tässä tämä rivitys hämää jotenkin aika paljon. Silmä ei ole tottunut lukemaan näin raskaita annotaatioparametrisointeja, joten jouduin lukemaan tämän useaan kertaan ennen kuin tajusin että kyseessä ei ole kokonaan oma, muusta irrallinen blokkinsa. Olisikohan tässä mahdollista esim. laittaa noita patheja johonkin omaan muuttujaansa (`frameConverterPaths` tms.,) jolloin koko annotaation määrittely mahtuisi yhdelle riville?

Reply:
Kuulostaa kyllä hieman turhalta epäsuoruudelta vaikka tämä onkin monirivinen 🤔 (olisi kuitenkin varmaan suoraan tämän luokkamääreen yläpuolella tuo polkumuuttuja). Muuallahan tällaisia multibindejä ei toki ole käytössä